core/src/main/scala/org/apache/spark/sql/collection/Utils.scala

```diff
@@ -368,6 +368,52 @@ object Utils {
     k
   }
 
+  /**
+   * Utility function to return a metadata for a StructField of StringType, to ensure that the
+   * field is stored (and rendered) as VARCHAR by SnappyStore.
+   * @param md
+   * @param size
+   * @return
+   */
+  def varcharMetadata(md: Metadata = Metadata.empty, size: Int = Constant.MAX_VARCHAR_SIZE):
```

Comment:
Can you reverse the order of arguments here? Users will normally not want to provide any explicit Metadata argument but may certainly want to specify some other max length, so let the size argument be the first one (and if users leave out second one then it will use Metadata.empty as above).